### PR TITLE
Region fixes

### PR DESCRIPTION
**Pull request by intOrfloat**

### CODE REVIEW

File: server/playerServer/playerserver.js [hunk 1]
```diff
@@ -662,7 +662,7 @@ var server = http.createServer(function (req, res) {
 		var uKey = parsedUrl.query.uKey;
 		var room = parsedUrl.query.room;
 		var regionId = parsedUrl.query.regionId;
-		var overrideOwner = parsedUrl.query.overrideOwner;
+		var overrideOwner = (parsedUrl.query.overrideOwner === 'true') ? true : false; //sent in as string.
```
Comment (lukasdietrich) [inline server/playerServer/playerserver.js]: `boolean ? true : false` is a bit verbose

Comment (intOrfloat) [inline server/playerServer/playerserver.js]: all things from parsedUrl.query that isnt an array acts like a string D'; so it literally sends a string with true or false in it.

Comment (lukasdietrich) [inline server/playerServer/playerserver.js]: nice ninja edit

Comment (intOrfloat) [inline server/playerServer/playerserver.js]: Alt Text

Comment (Squarific) [inline server/playerServer/playerserver.js]: memes

**Pull request merged**